docs/config.md

```diff
@@ -199,21 +199,28 @@ Obviously, you would only set one line, don't copy them all in!
 ## Editing diffs
 
 The `ui.diff-editor` setting affects the tool used for editing diffs (e.g.
-`jj split`, `jj amend -i`). The default is `meld`. The left and right
-directories to diff are passed as the first and second argument respectively.
+`jj split`, `jj amend -i`). The default is `meld`.
+
+`jj` replaces the following arguments:
```

Comment:
Here in and merge-tools below, I would now say "`jj` makes the following substitutions", since these don't have to be entire arguments.